catroid/src/main/java/org/catrobat/catroid/stage/StageActivity.java

```diff
@@ -391,6 +391,7 @@ protected void onDestroy() {
 		CameraManager.getInstance().stopPreviewAsync();
 		CameraManager.getInstance().releaseCamera();
 		CameraManager.getInstance().setToDefaultCamera();
+		ProjectManager.getInstance().setSceneToPlay(ProjectManager.getInstance().getCurrentScene());
```

Comment:
I don't know.
I've just rebased to the current develop, where this code is already merged. 
This PR goes to the feature/NOLB-branch, so only the last commit should be worthwhile to review.